src/main/java/com/google/codeu/servlets/ClinicDataServlet.java

```diff
@@ -25,13 +25,15 @@ public void init() {
     Scanner scanner = new Scanner(getServletContext().getResourceAsStream("/WEB-INF/clinics-data.csv"));
     while(scanner.hasNextLine()) {
       String line = scanner.nextLine();
-      String[] cells = line.split(",");
-
+      String regex = ",(?=(?:[^\"]*\"[^\"]*\")*[^\"]*$)";
```

Comment:
Use a class const for the regex.

private static final String SPLIT_REGEX_CONST = ",(?=(?:[^\"]*\"[^\"]*\")*[^\"]*$)"